app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    (person&.admin? || person&.presentations&.include?(record)).present?
```

Comment:
For documentations sake the resolution to this conversation is listed here:

https://www.pivotaltracker.com/story/show/142711001